pkg/kubectl/cmd/util/helpers.go

```diff
@@ -331,6 +331,10 @@ func AddValidateFlags(cmd *cobra.Command) {
 	cmd.Flags().String("schema-cache-dir", fmt.Sprintf("~/%s/%s", clientcmd.RecommendedHomeDir, clientcmd.RecommendedSchemaName), fmt.Sprintf("If non-empty, load/store cached API schemas in this directory, default is '$HOME/%s/%s'", clientcmd.RecommendedHomeDir, clientcmd.RecommendedSchemaName))
 }
 
+func AddRecursiveFlags(cmd *cobra.Command) {
+	cmd.Flags().Bool("recursive", false, "If true, process files recursively when given a directory")
```

Comment:
I'd suggest a shorthand uppercase version of the flag, as supported in major unix commands like cp, ls, chmod, etc:

```
-R, --recursive
```